org.eclipse.m2e.core/src/org/eclipse/m2e/core/internal/embedder/ReadonlyMavenExecutionRequest.java

```diff
@@ -522,6 +522,16 @@ public MavenExecutionRequest setUseLegacyLocalRepository(boolean useLegacyReposi
     return request.setUseLegacyLocalRepository(useLegacyRepository);
   }
 
+  // @Override // Support compilation against Maven 3.8
```

Comment:
@laeubi in the CI all M2E bundles build against the maven 3.8.6 runtime version and therefore compilation fails if these new methods are annotated with `@Override`. I assume that's intended and you probably don't want to raise the lower bound to Maven 3.9, don't you?

Reply:
OK this will of course cause problems in the implementation since the new methods are also called.

So we either have to raise the lower bound or use reflection here.